packages/blockchain-wallet-v4-frontend/src/data/components/sendXlm/sagas.js

```diff
@@ -91,8 +91,10 @@ export default ({ api, coreSagas }) => {
           payment = yield payment.to(value)
           // Do not block payment update when to is changed w/ destinationAccount check
           yield put(A.paymentUpdatedSuccess(payment.value()))
-          // After updating payment success check if destinationAccount exists
+          // check if destination exists
```

Comment:
This specifically needs to happen _after_ updating payment success. If thats clear enough then fine to remove